packages/faceted-search/src/components/FacetedToolbar/FacetedToolbar.scss

```
@@ -0,0 +1,50 @@
+$toolbar-height: 5.5rem;
+
+.tc-faceted-toolbar {
+	align-items: center;
+	display: flex;
+	justify-content: space-between;
+	min-height: $toolbar-height;
+	position: relative;
+	width: 100%;
+
+	&-buttons {
+		background: $alto;
+		border-radius: $padding-large $padding-large $padding-large $padding-large;
+		display: flex;
+		margin-right: $padding-large;
+		position: absolute;
+		margin-top: $padding-normal;
+		right: 0;
+		top: 0;
+
+		&-basic,
+		&-advanced {
+			&.tc-button-selected {
+				background: $st-tropaz;
+				border-radius: $padding-large;
+				color: white;
+			}
+			background: $alto;
+			color: black;
+			min-height: 25px;
+			height: 25px;
+			font-size: 10px;
+			> span {
+				height: inherit;
+			}
+			&:active,
+			&:focus,
+			&:hover {
+				background: $st-tropaz;
+			}
+		}
+
+		&-advanced {
```

```suggestion
		&-advanced, &-advanced.tc-button-selected {
```
There is a visual bug when advanced action is selected: radius is add to action, here is proposition of fix